### DISCUSSION TITLE
796

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#972] [Comments]
# 796

요즘 Todo App을 오프라이트로 바꿔서 사용해보고 있다.기존에 TickTick을 사용했는데, TickTick이 가지는 여러가지 장점이 있지만, 아직 통합 기능이 부족하다.분산된 여러 업무 도구들이 자동으로 연계 되어 굳이 내가 별도로 기록할 일이 없도록 하는 것이 필요한데, 그런 부분에서 TickTick은 현재까지 (2024.08.06) 구글 캘린더 통합만 지원한다.반면 오프라이트의 경우 슬랙, 지메일, 구글 캘린더, Zapier 등 여러 플랫폼을 지원하고 있어 파편화 된 업무 정보를 한 곳에서 볼 수 있다.물론 이런 업무 도구에 쌓인 정보들은 회사의 중요 정보가 되기도 하기 때문에 공개 범위를 적절하게 사용하는 것이 좋다.다만, 여전히 TickTick도 함께 사용중인데, 그 이유는 모바일 기기에서의..

https://jojoldu.tistory.com/796

**Comment by jws1837** [comment]
Ticktick, 오프라이트 모두 첨들어보는데 유용해보이네요. 기회되면 나중에 써봐야겠습니다!
